### PR TITLE
Added Zeronights 2016 materials

### PR DESCRIPTION
**Pull request by alexxn**

### CODE REVIEW

**Comment by alexxn** [comment]
@PaulSec ping

**Comment by tunnelsup** [comment]
@alexxn the link you gave just has a bunch of pdfs. Do you have a link to the video recordings?

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Conference | Location | Date
 |[BSides Columbus 2016](https://www.irongeek.com/i.php?page=videos/bsidescolumbus2016/mainlist) | Columbus, OH | Jan 18, 2016 |
 |[Shmoocon 2016](https://archive.org/details/shmoocon-2016) | Washington, DC | Jan 15, 2016 |
 |[Shmoocon Firetalks 2016](https://www.youtube.com/playlist?list=PLNhlcxQZJSm_gTMJmDNjUcN15HK6-p4H4) | Washington, DC | Jan 15, 2016 |
+|[Zeronights 2016](https://2016.zeronights.org/conference-materials/presentations/ | Mosow, Russia | Nov 17-18, 2016 |
```
Comment (thypon) [inline README.md:33]: sed 's|Mosow|Moscow|g'

Comment (PaulSec) [inline README.md]: Hey there, 

The entry is still breaking the actual layout (there's a parenthesis breaking the table layout) but also, as @tunnelsup suggested below, the link is only about the PDFs, let's wait for the Youtube channel and we'll add them there. 

Cheers,

**Comment by PaulSec** [comment]
Hey there, 

the entry is still breaking the table layout (a parenthesis is breaking it) and the link, as @tunnelsup suggested is only the PDFs. Let's wait for the videos to be up and we'll add it at that point. 

Cheers and thanks for the contribution,

**Comment by alexxn** [comment]
Unfortunately videos in english are not available yet and probably won't be

**Comment by PaulSec** [comment]
Argh, too bad :/ 
Let's see and wait.

**Pull request closed without merging**